controller/array_action/array_mediator_ds8k.py

```diff
@@ -617,27 +617,41 @@ def get_iscsi_targets_by_iqn(self, host_name):
 
     def get_array_fc_wwns(self, host_name):
         logger.debug("getting the connected fc port wwpns for host {} from array".format(host_name))
-
+        api_host = self._get_api_host(host_name)
+        if api_host is None:
+            return []
```

Comment:
I don't think this should happen. why not keep raising `HostNotFoundError` as before?

Reply:
I think `get_array_fc_wwns` still doesn't raise `HostNotFoundError`

Reply:
[interface indicates that it raises None.](https://github.com/IBM/ibm-block-csi-driver/blob/8a1076ba5e079dffac2f867a69ab797bb4477f32/controller/array_action/array_mediator_interface.py#L301)

Reply:
I think it means the caller should not expect any specific exception (e.g. most if not all mediator methods can cause a storage error).
but regardless of current interface, I don't see the point in continuing the ControllerPublish -> NodeStage flow if we don't pass any fc targets in the publish context